src/native-automation/utils/string.ts

```diff
@@ -1,11 +1,12 @@
 import Protocol from 'devtools-protocol';
 import GetResponseBodyResponse = Protocol.Network.GetResponseBodyResponse;
 import HeaderEntry = Protocol.Fetch.HeaderEntry;
+import { decodeBufferToString, encodeStringToBuffer } from 'testcafe-hammerhead';
 
-export function getResponseAsString (response: GetResponseBodyResponse): string {
-    return response.base64Encoded
-        ? Buffer.from(response.body, 'base64').toString()
-        : response.body;
+export function getResponseAsString (response: GetResponseBodyResponse, contentType = 'charset=utf-8'): string {
```

Comment:
let's remove this default value.
Lets write more explicit.
If we have content type then use `decodeBufferToString`.
If we do not have for some reason lets use default bufferBody.toString.
Smth like this:
```
return contentType ? decodeBufferToString(bufferBody, contentType) : bufferBody.toString()
```